README.md

```diff
@@ -1,7 +1,80 @@
-# Apache SkyWalking Agent Test Tool Suits
+Apache SkyWalking Agent Test Tool Suite
+==========
 
+<img src="http://skywalking.apache.org/assets/logo.svg" alt="Sky Walking logo" height="90px" align="right" />
+
+[![Twitter Follow](https://img.shields.io/twitter/follow/asfskywalking.svg?style=for-the-badge&label=Follow&logo=twitter)](https://twitter.com/AsfSkyWalking)
+
+![CI](https://github.com/apache/skywalking-nginx-lua/workflows/CI/badge.svg?branch=master)
+
+
+[**SkyWalking**](https://github.com/apache/skywalking) Agent Test Tool is a tremendously useful test tools suite in a wide variety of languages of `Agent`. 
+Includes `mock collector` and `validator`. The `mock collector` is a SkyWalking receiver, likes OAP server. 
+
+The `mock collector` is responsible for mocking the SkyWalking OAP receiver to collect data from the agent. It will collect data from Agent reported by `GRPC`. 
```

Comment:
Remove `will`, Does this support HTTP? `it receives data from agent through GRPC and HTTP`?